benchmarks/backend_request_func.py

```diff
@@ -388,6 +390,13 @@ def remove_prefix(text: str, prefix: str) -> str:
     return text
 
 
+def get_tokenizer(
+    pretrained_model_name_or_path: str, trust_remote_code: bool
+) -> Union[PreTrainedTokenizer, PreTrainedTokenizerFast]:
+    return AutoTokenizer.from_pretrained(pretrained_model_name_or_path,
+                                         trust_remote_code=trust_remote_code)
```

Comment:
Could you modify this with the following so that it can at least supports tokenizer from modelscope? Thanks!
https://github.com/vllm-project/vllm/blob/e2b85cf86a522e734a38b1d0314cfe9625003ef9/vllm/transformers_utils/tokenizer.py#L68-L83

Reply:
ok

Reply:
Hi @ywang96 done. Thanks for the review!